src/CodeCoverageExtension.php

```diff
@@ -12,18 +10,24 @@
  * that was distributed with this source code.
  */
 
+declare(strict_types=1);
```

Comment:
Can you elaborate why you moved that statement?

Reply:
PHPCsFixer from GrumpPHP does this.